modules/core/test/test_lpsolver.cpp

```diff
@@ -158,13 +160,12 @@ TEST(Core_LPSolver, issue_12343)
                                        3., 1., 2., 2., 3.,
                                        4., 4., 0., 1., 4.,
                                        4., 0., 4., 1., 4.);
-
-    Mat z;
-    int result = cv::solveLP(A,B,z);
-    EXPECT_EQ(result, 0);
-
-    Mat etalon_z = (cv::Mat_<double>(4, 1) << 16./37., 15./37., 0.0, 24./37.);
-    ASSERT_LT(cvtest::norm(z, etalon_z, cv::NORM_L1), 1e-12);
+    for (int i = 1; i < 1e5; i*=5)
+    {
+        Mat z;
+        int result = cv::solveLP(A,B*i,z);
```

Comment:
In linear task both A/B should be multiplied to keep the same result.

Reply:
No it's not true. `5x+10y < 25` is the same as `x+2y < 5`. So mathematically it's correct test, but the idea of test is not obvious. Is the provided matrix is bed defined for Simplex Method?